fecgov/openFEC
# Feature/4063 add endpoint caluclate total cc

## Summary

- Resolves #4063 

- add communication costs totals endpoint to sum over committees on support or oppose

## How to test the changes locally

- checkout this branch
- `pytest`
- run local api and see new endpoint in swagger ui--see test it out to see that totals are given as expected.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  `/communication_cost/by_candidate/totals/` endpoint


## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
| | #4097 |
| | #4096 |
